fbw-a32nx/src/wasm/fadec_a320/src/EngineControl.h

```diff
@@ -85,6 +87,8 @@ class EngineControl {
   double imbalance;
   int engineImbalanced;
   double paramImbalance;
+  double prevEngineMasterPos[2] = {0};
+  bool prevEngineStarterState[2] = {0};
```

Comment:
I'm not a C++ expert, but I think it would be nicer to initialise them like this.

```suggestion
  double prevEngineMasterPos[2] = {0, 0};
  bool prevEngineStarterState[2] = {false, false};
```

Reply:
done